### ISSUE TITLE
Warlock Buffs (BB #728)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#728]
This issue was migrated from bitbucket.
**Original Reporter:** realyzee
**Original Date:** 01.11.2014 20:26:59 GMT+0000
**Original Priority:** minor
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728

<hr>

Hey, 
One of our players highlighted you can stack your Demon Skin and Fel Armor spells. I mean you can both of it on yourself at the same time.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** realyzee
**Original Date:** 01.11.2014 20:28:59 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13362215

<hr>

You can also stack Demon Skin with Demon Armor, etc..

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 01.11.2014 22:05:05 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13362920

<hr>

Latest DB?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 02.11.2014 01:01:08 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13363910

<hr>

https://bitbucket.org/oregon/oregoncore/commits/9822a2ca271c9d81df70f912694318cdc00b6f8c

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zurr
**Original Date:** 04.11.2014 10:35:41 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13407854

<hr>

yep,one more  spell : Demon skin.
Atm we have 2 another warlock spells in spell stack rules : Demon armor and Fell armor.
Demon skin not included

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 04.11.2014 12:05:03 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13409822

<hr>

Demon skin does not stack with demon armor and fel armor?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zurr
**Original Date:** 04.11.2014 15:10:35 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13413914

<hr>

its stacks on clean core i think.
Blizzlike: only one armor can be active at same time.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 04.11.2014 15:14:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13414005

<hr>

Try this and let me know

```
#!sql
DELETE FROM spell_group WHERE id = 1018 AND spell_id = 687;
INSERT INTO spell_group (id, spell_id) VALUES
(1018, 687); -- Demon Skin
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zurr
**Original Date:** 04.11.2014 15:27:16 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13414305

<hr>

its work properly now

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 04.11.2014 15:28:47 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13414345

<hr>

Sweet, I'll push today.

Edit: Thankyou by the way :)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zurr
**Original Date:** 04.11.2014 15:34:33 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13414482

<hr>

No problem  : )

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 04.11.2014 16:58:02 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/728/#comment-13416466

<hr>

Resolved in 6b282811169f09d41649ffd7cf18f0f89b5b4e97
